App.tsx

```diff
@@ -474,11 +476,24 @@ const BaseApp: React.FunctionComponent<{
     );
   }
 
+  const linking = {
+    prefixes: [prefix, FormatCenterWebsite(avalancheCenterId as AvalancheCenterWebsiteSchema) + '/observations/#/view/'],
```

Comment:
I need to look more into how this works in RN, but you might not need this here. This seems like it should only be the base URLs that the app accepts

Reply:
This is documented as the configuration to map paths to screens: https://reactnavigation.org/docs/configuring-links#mapping-path-to-route-names

Reply:
Oh sorry I meant the `/observations/#/view` might not be needed. In those docs it seems like just the base URLs are being used in the `prefixes`

Reply:
I see what's going on here now

Reply:
> n.b. @yuliadub making these prefixes dynamic based on the currently-focused center means that someone can't open a link to another center's observations page if they aren't already looking at that center's data - is that what we want?

I went back and forth on this....i.e. do we hard code a list in there or load it dynamically based on the center you are in...welcome all thoughts and opinions on this since I dont 100% know what is going to work best (maybe once we figure out the user experience we want with opening a shared obs not from your center it could help guide us too)